packages/react-server/src/ReactFlightDirectives.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
This file seems unnecessary indirection by itself. Better putting this and the Dispatcher into a separate file.

But let's make sure we call everything FlightServer or FlightClient if they're specific to only one side. E.g. this one is only for Server.

It might be worth creating a folder for all these new files because this is the first time (other than Cache which should move too) we have helper files in `react-server/src`. Everything else is kind of top level things.

Maybe `react-server/src/flight-server/` and `react-server/src/flight-client`.

Reply:
I guess `react-server` implies it's for Client but since we kind of use global naming convention for files.

Reply:
eliminated all of these new helper files